templates/general.js

```diff
@@ -85,17 +85,29 @@ ${name};
     return `
 {
   "name": "${moduleName}",
+  "title": "${moduleName.split('-').map(word => word[0].toUpperCase() + word.substr(1)).join(' ')}",
```

Comment:
I'm wondering if we need this property here or if we could prepare this in the `templates/android.js` file instead?

Reply:
Actually, I placed it into `package.json` on purpose. It may only be used by the Android maven config at this time, but for the sake of consistency I think it's good to have it here so that all config values that feed into the maven config can be changed in one place.